.env

```diff
@@ -0,0 +1 @@
+VERIFY_EVENT_SYSTEM_DATABASE_SCRIPTS_LOCATION=../verify-event-system-database-scripts
```

Comment:
So this isn't using direnv, and we're not using pyenv because this is going to be interpreted by a bash script... so what is this file going to be used by?

If it's just an example, it should be called `example.env` (ie. making it a hidden file isn't especially beneficial, because nobody will see it there, and then they won't know what it contains).

But anyway, I don't think it quite works, having a relative path in here. If it's set, your script isn't interpreting it relative to anything as far as I can see, so we're back to it being important to run from the correct directory?

Reply:
It's the default value that will be used by `docker-compose.yml` if not present in the environment. I mention in commit message but forgot amend PR description.

I'll put a comment in the file too, so it's clear what its for.